cli/package.json

```diff
@@ -38,7 +38,7 @@
     "dayjs": "^1.10.4",
     "debug": "4.3.2",
     "eventemitter2": "^6.4.3",
-    "execa": "^5.0.0",
```

Comment:
in Node 14, execa@5 + mock-spawn have some incompatibility - downgrading execa fixes